test/extended/cluster/metrics.go

```diff
@@ -0,0 +1,56 @@
+package cluster
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"time"
+)
+
+const (
+	marker_name string = "cluster_loader_marker"
```

Comment:
What is this for?

Reply:
This is for the 3rd part tool to know that this line is for parse ... hard coded for now .. could be specified in the config file if needed in the future.

Reply:
I was thinking that there would be more of a log line marker, so before (and maybe after) the json is printed it looks something like...
```
Mar  6 12:12:16.482: INFO: etc, etc...
Mar  6 12:12:16.723: INFO: etc, etc...
<<< BEGIN METRICS OUTPUT >>>
{ "json": true }
<<< END METRICS OUTPUT >>>
```

That's just a conceptual example of what I was thinking. That way the 3rd party tool just looks for the `BEGIN` marker and then looks for json after that point?
What do you think?

Reply:
(12:37:29 PM) hongkliu: sejug: for now we have only one line/object to print ... what if we have more lines? and how can we ensure there is no other calling of fmt ... so that they wont be inserted between our markers?
(12:38:14 PM) hongkliu: sejug: another advantage to do it my way ... we can use grep to filter out the msg from the stout ... just want to say it out before i change the code

my 2 cents:
If the line is self contained each line ... it is easier for the 3rd party to parse.

Reply:
Sounds good, lets keep it as is.